### PR TITLE
add ICD10 grouping term mappings to Mondo terms

### PR DESCRIPTION
**Pull request by nicolevasilevsky**

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -46382,6 +46450,7 @@ def: "A metabolic disorder characterized by abnormal blood glucose levels." [NCI
 synonym: "disorder of glucose metabolism" EXACT [NCIT:C53655]
 synonym: "glucose metabolism disorder" EXACT [NCIT:C53655]
 xref: DOID:4194 {source="MONDO:equivalentTo"}
+xref: ICD10CM:E15-E16 {source="MONDO:subClassOf", source="https://orcid.org/0000-0001-5208-3432", source="https://orcid.org/0000-0002-4142-7153"}
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:46453]: We say here that "disorder of glucose metabolism" is a subClassOf "Other disorders of glucose regulation and pancreatic internal secretion". The key being "other here" - are we certain that is what we want to do?

**Review by matentzn — APPROVED** [review]
Formally, this is great. Mapping wise, I have a few questions, but I think what we need to do organise a tutorial and creating mappings: In particular, terms that involve the word "other" should be taking as a restriction on the concept, which means they can only be MONDO:superClass. For example, in the case of `MONDO:PNS disease` and `ICD10:Other PNS disease`, `MONDO:PNS disease` can only be a parent of ICD10. If though the ICD10 class has another disjunctive clause like "ICD10:Other PNS disease or Psychiatric disorder", then, unless `Psychiatric disorder` is a child of `PNS disease`, there cannot be _any_ relation between the Mondo and ICD10 terms (aside from "related").

**Comment by nicolevasilevsky** [comment]
@matentzn I love the idea of organizing a tutorial to cover mappings, as I am unsure about all of this!

For now, do you think we should just ROBOT in the equivalence mappings, since the subclassOf and superclassOf mappings may not be correct?

I am planning to run a release tomorrow.

**Comment by nicolevasilevsky** [comment]
The examples that you gave above are things @sabrinatoro and I discussed and were unsure about.

**Comment by nicolevasilevsky** [comment]
> For example, in the case of MONDO:PNS disease and ICD10:Other PNS disease, MONDO:PNS disease can only be a parent of ICD10.

I thought this is what we did, no?

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:46453]: this was a particularly confusing case, so no, I am not certain!

Comment (matentzn) [reply src/ontology/mondo-edit.obo:46453]: I think I am neither.. We can discuss this on Wednesday!

**Comment by matentzn** [comment]
This does not have to merged for the release. I am fine with having it in an unmerged state and work on it a bit more calmly.

**Comment by nicolevasilevsky** [comment]
Ok, I converted it to a draft.

**Comment by matentzn** [comment]
I fear I do not have the bandwidth to see merging this PR through.

@nicolevasilevsky can you perhaps develop a more conservative approach to get these mappings merged in? Maybe start with all the ones we are certain are good, and than work on the others one by one? On the other hand, we can also simply add everything, and hope boomer howls if something appears strange?

**Comment by nicolevasilevsky** [comment]
Sure, let me think about this. I'll add this to my to do list to figure out how to best approach this.

**Comment by nicolevasilevsky** [comment]
close this - redoing on another PR

see: https://github.com/monarch-initiative/mondo/issues/4536

**Pull request closed without merging**